### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by onerinas**
Read me update with code for previously used numeric sequence

Suggestion reference: https://github.com/norman/friendly_id/issues/602#issuecomment-54272912

Code reference: https://github.com/norman/friendly_id/issues/576#issuecomment-47485811

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,28 @@ The most important changes are:
   sequence to differentiate conflicting slug, but rather a UUID (e.g. something
   like `2bc08962-b3dd-4f29-b2e6-244710c86106`). This makes the
   codebase simpler and more reliable when running concurrently, at the expense
-  of uglier ids being generated when there are conflicts.
+  of uglier ids being generated when there are conflicts. 
+* If you need previously used numeric sequence for some reason:
```
Comment (parndt) [inline README.md]: I think we should remove phrases like `for some reason` as this potentially alienates those with this requirement by making their use case seem trivial or nonsensical.  Perhaps I'm reading too far into it, but if I do then perhaps others will too. :blue_heart:

**Comment by onerinas** [comment]
should we consider reframing this sentence 
or
 add a sentence like "the old behaviour is unsupported by the new code base and discouraged, but if you really think you need it check issues #430 #480, #576 and #602 for discussion and workarounds". as mentioned in https://github.com/norman/friendly_id/issues/602#issuecomment-54272912 
or 
do nothing :smile:

File: README.md (hunk 1 above)
Comment (onerinas) [inline README.md]: should we consider re-framing this sentence to "If you want to override UUID with numeric sequence: "
or
add a sentence like "the old behaviour is unsupported by the new code base and discouraged, but if you really think you need it check issues #430 #480, #576 and #602 for discussion and workarounds". as mentioned in #602 (comment) 
or 
do nothing :smile:

Comment (kimrgrey) [inline README.md]: It looks a little bit outdated now. May be it's better to add some documentation about `SequentiallySlugged ` instead of it?

**Comment by kimrgrey** [comment]
Well, I'll close this PR because we should add some documentation  about module `:sequentially_slugged` instead of it. Thank you for your contribution, anyway! We really appreciate it! :heart:

**Pull request closed without merging**